src/main/java/salesforce/selectors/XPaths.java

```diff
@@ -0,0 +1,34 @@
+package salesforce.selectors;
+
+public enum XPaths {
```

Comment:
I think you don't use your enum Xpath, for me is a great solution, but I think this will be better if you replace them
and use @FIndBy with a PageFactory

Reply:
I already deleted it.